.pre-commit-config.yaml

```diff
@@ -146,9 +146,7 @@ repos:
             --exclude=libparsec_bindings_electron,
             --,
             --deny=warnings,
-            --deny=clippy::self_named_module_files,
```

Comment:
Nop, we keep that

Reply:
This lint is broken, @touilleMan can't have his module named `types` because `libparsec_types` already uses this name

Reply:
So it's a bug, do we have opened an issue on clippy ?

Reply:
Reported: https://github.com/rust-lang/rust-clippy/issues/10271

Reply:
:+1: